common/lib/api-client/middleware/request-include.js

```diff
@@ -19,7 +19,11 @@ module.exports = {
       include = include.sort().join(',')
     }
 
-    req.params.include = include
+    if (include) {
```

Comment:
Also this could do with a unit test update?

Reply:
Yep. That would have clarified what the purpose of the change was.

And if the tests continue passing after the change to make the code good, they can carry on passing if someone makes the change back the other way...